highfidelity/hifi
# Beta Release 13 - Includes up to Developer Release 5026

- Add audio peak limiter to audio mixer
- Fix the jittering reflection bug and more Normal improvements
- vhacd-util improvements
- Fix domain max capacity to only count users (not scripted agents)
- Make navigating directories in file browser dialog possible without double-clicking
- Fix Avatar collision sounds triggering all the time
- Make the application of redundant physics update idempotent to improve action glitches
- Fix crash on shutdown of SteamVR
- Fix crash on click on non-visible web-entity 
- Update the build docs for a better OpenSSL security posture.
- Do not send stale vive data as valid after deactivating
- Save toolbar position when moving it only when stopped moving it
- Add leap hands to /system
- Replace the linear interpolation embedded in the .WAV/.RAW loader with high quality polyphase resampling.
- Fix bad convex hull simplification
- Revert "Update the build docs for a better OpenSSL security posture."
- Eliminate file IO contentions for settings
- Removed innocuous "AvatarData packet size mismatch" warning
- Add version number to avatar recording frame
- Add Clipboard.getContentsDimensions() JavaScript method
- Eliminate extraneous writes for unchanged values
- Add Entities.getChildrenIDs() method
- Camera steers avatar when walking
- Fixed constant turn of 22.5 degrees in comfort mode
- Add "Load Defaults" button to Running Scripts dialog and rename "Stop All" to "Remove All"
- Select/Disable plugins via command line args
- Fix OS X warning
- Fix colors of buttons in Asset Browser dialog
- AVX2 optimized audio resampler 
- New home content
- Corrected Oculus Touch coordinate system so that hands track properly
- Fix for models exported from Daz3D
- Expose haptics to JS
- Game pads work after falling asleep and being reactivated
- Add support for ambient occlusion texture and lightmap texture for fbx exported from Blender
- Fix height of Load Defaults button in Running Scripts dialog
- Added MyAvatar.hmdLeanRecenterEnabled property used to disable the 'room-scale' avatar re-centering code. Disabling this can prevent sliding when the avatar is supposed to be sitting or mounted on a stationary object. 
- Add basic metadata to domains
- Attempt to get better logging from pure virtual call crashes
- Use shared CPUDetect.h for CPU detection
- Re-instate domain API connection info refresh during failure, fix localhost connection after downgrade
- Rework domain-level permissions 
- Fix some permissions-grid bugs
- Fix import of invalid JSON file failing silently
- Fix model export
- Update BUILD_OSX.md
- Fix avatar entity locking
- Fix problem related to server-side extrapolation of children of avatars
- Fix image2d overlay color property
- Fix tabbing behavior in edit.js
- Exposed orientation and eye position to procedural entity shaders
- More verbose messaging about incorrect case in entity types
- Reset Vive grip buttons to 0 if not pressed
- Only signal once on a protocol mismatch with DS
- Fixing issues with unclosed groups in settings persistence

# **QA testing**  

http://rawgit.com/cozza13/QAtesting/master/dynamicrc.js

Full **Smoke test** - **HMD RC test** with **Eye tests** and **RC test**

This PR has a protocol change. Therefore you will only see content on you updated locahost, dev-playa and dev-demo. I have updated the server stats to pull from those locations
